action/lib/Database.js

```diff
@@ -60,4 +63,39 @@ module.exports = function(dbURL, dbName) {
                 });
             })
     };
+
+    this.getTriggerAssignment = function(workers) {
+        // a map between available workers and their number of assigned triggers
+        // values will be populated with the results of the assignment view
+        console.log(`Available workers: ${JSON.stringify(workers, null, 2)}`);
```

Comment:
This log statement seems misleading since `worker0` is used if the workers parameter has an empty array as an argument.

Reply:
Referring to this line below: `var assignment = workers[0] || 'worker0';`.